02-overview-surveys.Rmd

```diff
@@ -149,39 +150,39 @@ To avoid this issue, researchers should consider these possibilities and adjust
 
 Researchers can then code the responses from the open-ended box and get a better understanding of the respondent's choice of preferred pet. Interpreting this question becomes easier as researchers no longer need to qualify the results with the choices provided.
 
-This is a very simple example of how the presentation of the question and options can impact the findings. More complex topics and questions will need researchers to thoroughly consider how to mitigate any impacts from the presentation, formatting, wording, and other aspects. As survey analysts, reviewing not only the data but also the wording of the questions is crucial to ensure the results are presented in a manner consistent with the question asked.
+This is a very simple example of how the presentation of the question and options can impact the findings. More complex topics and questions will need researchers to thoroughly consider how to mitigate any impacts from the presentation, formatting, wording, and other aspects. As survey analysts, reviewing not only the data but also the wording of the questions is crucial to ensure the results are presented in a manner consistent with the question asked. 
 
 ## Data Collection {#overview-datacollection}
```

Comment:
Perhaps under Data Collection, we can talk a little bit about data entry and quality?